types/mui-image/index.d.ts

```diff
@@ -0,0 +1,35 @@
+// Type definitions for mui-image 1.0
+// Project: https://github.com/benmneb/mui-image
+// Definitions by: benmneb <https://github.com/benmneb>
+//                 Natalia <https://github.com/CodeMeNatalie>
+// Definitions: https://github.com/DefinitelyTyped/mui-image
+
+import * as React from 'react';
+
+interface MuiImageProps {
+    alt?: string;
+    bgColor?: React.CSSProperties['backgroundColor'];
+    className?: string;
+    distance?: string | number;
+    duration?: number;
+    easing?: React.CSSProperties['transitionTimingFunction'];
+    errorIcon?: boolean | React.ReactNode;
+    fit?: React.CSSProperties['objectFit'];
+    height?: React.CSSProperties['height'] | number;
+    iconWrapperClassName?: string;
+    iconWrapperStyle?: React.CSSProperties;
+    onError?: (...args: any[]) => any;
+    onLoad?: (...args: any[]) => any;
+    position?: React.CSSProperties['position'];
+    shift?: 'left' | 'right' | 'top' | 'bottom' | false | null;
+    shiftDuration?: number;
+    showLoading?: boolean | React.ReactNode;
+    src: string;
+    style?: React.CSSProperties;
+    width?: React.CSSProperties['width'] | number;
+    wrapperClassName?: string;
+    wrapperStyle?: React.CSSProperties;
```

Comment:
```suggestion
    alt?: string | undefined;
    bgColor?: React.CSSProperties['backgroundColor'] | undefined;
    className?: string | undefined;
    distance?: string | number | undefined;
    duration?: number | undefined;
    easing?: React.CSSProperties['transitionTimingFunction'] | undefined;
    errorIcon?: boolean | React.ReactNode | undefined;
    fit?: React.CSSProperties['objectFit'] | undefined;
    height?: React.CSSProperties['height'] | number | undefined;
    iconWrapperClassName?: string | undefined;
    iconWrapperStyle?: React.CSSProperties | undefined;
    onError?: (...args: any[]) => any | undefined;
    onLoad?: (...args: any[]) => any | undefined;
    position?: React.CSSProperties['position'] | undefined;
    shift?: 'left' | 'right' | 'top' | 'bottom' | false | null | undefined;
    shiftDuration?: number | undefined;
    showLoading?: boolean | React.ReactNode | undefined;
    src: string;
    style?: React.CSSProperties | undefined;
    width?: React.CSSProperties['width'] | number | undefined;
    wrapperClassName?: string | undefined;
    wrapperStyle?: React.CSSProperties | undefined;
```
would be nice to have support for TS4.4 exact optional property types here:
https://www.typescriptlang.org/play?exactOptionalPropertyTypes=true&ts=4.4.2&ssl=1&ssc=1&pln=2&pc=1#code/PTAEHUEsBcAtQKYA8CGBjaB5ADtSB7AOxQBsAFAJ32wQugE8AVemgZ0WICMSEATAGlDMaAZTQVIuALAAoEKFgp2raBIy1QFAK492KKlsK9QAd0XRQ9fFtBoUhUKwQX7oaniKlZ87FRp16UGh8ADpZbzAAMXwKRFQAW2weQThIdkhCaFoAM3QEUF4ENBJ9BHY4FAs4Wny00BQI0F9qWgZTWEg0eDsHTnyifuygk3xHVQyAc1YALlAAcl59AGs5t1i5kkgJ2Gg5xpjQGEdYaxJjQnwLPsOHardOACsi6DCZWQysily0fIBVJwoABEELkdNB2ABvWSgUDyRiwfIoThOQg-NxDFCgABupC0+QoCF8ZQQmXYc1Y9BUCHiexkMLQ+BIMXh1IQmCxtAkhQA-LMAESLChLPmgAA+oD5m220D5AG5ZABfcJyMBQODWKoddjZEoTDhIngCIIIgn1U1wAn5HEkPHsMydeBWGw9RpOCwAAwZTIoLPibI5FC5CHdQVGfX5guFgklWx2IvsxndhkK2QyfHdr0aImceEIeru1rxodASaMILTvBDJkgJBI9Vr+BMoHi+BUoAAUigcWIJLhNIY8H7Gl0ikttQc7sg0llUflgqBcjXBPauod2CDss9IBySIFciQKa95AAJRsIAMpE0Icn1BdaABe97TrHKo1S7D6iixBAoISECIKctCBgAhCEabIDk9RlmQRP12U5SBChDAB9Xh8DKQg5mgZCbmKLRChLZMgPTTMVX-fIdRQPV3WQdAsFwUDSEoFoAmEMoQ3iFAljKIQWAQHtJBcNA0C0ChKgQXdGk46BVzuFMMhAogmiob9CmMJR6gcdxGLrZp-AYUjCmKUo71RDwHAmZx-lobNoFzKYAAoAEpZmsoEQRQMFWHlN4ZAZQg2zdez2AAXlASzoDc2zgucnygsmVgQi9GDWXgwNEPyMKBWWOVZHivNEuSn1YP9BCCKyqU4x8xoACEtAsC5oGmPKcwSpLoOK1KAyDUAwqI1NCD4aqyMoH9izuSjqNojAcHMpi-FaJg+NYd1LzqEwNMappW1YSBuAQEIgA